src/styles/testribbon.scss

```diff
@@ -0,0 +1,76 @@
+.corner-ribbon .wrapper {
+  background: transparent;
+  height: 140px;
+  width: 35px;
+  position: absolute;
+  bottom: -34px;
+  right: 19px;
+  transform: rotate(45deg);
+  overflow: hidden;
+  z-index: 32;
+  -webkit-backface-visibility: hidden;
+
+  .container {
```

Comment:
Too generic class name, can easily be overriden unintentionally by other styles on user's webpage. Remember that booking.js isn't isolated from the rest of the webpage that it's rendered on.

Reply:
roger!